op-e2e/e2eutils/secrets.go

```diff
@@ -189,3 +192,14 @@ func (a *Addresses) All() []common.Address {
 		a.Mallory,
 	}
 }
+
+func (s *Secrets) AccountAtIdx(idx int) *ecdsa.PrivateKey {
```

Comment:
perhaps `uint` because negative values don't make sense here?

Reply:
It's common to use `int`s in Go for indices that don't make sense to be negative. So I just followed that convention. E.g. see stdlib, it uses `int` indices everywhere. The user of this API is usually an expert who wouldn't put in negative numbers. So I'd just leave it as is.